.github/workflows/linux_build.yml

```
@@ -70,6 +70,27 @@ jobs:
             ${{ runner.os }}-rust-v1-assembleRelease
             ${{ runner.os }}-rust-v1
 
+      - name: Bazel cache
+        id: bazel-cache
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.cache/bazel
+          key: ${{ runner.os }}-v1-bazel-cache-${{ hashFiles('rslib-bridge/anki/defs.bzl') }}
+          restore-keys: |
+            ${{ runner.os }}-v1-bazel-cache
+
+      - name: Node module cache
+        id: node-module-cache
+        uses: actions/cache@v2
+        with:
+          path: |
+            rslib-bridge/anki/node_modules
+          key: ${{ runner.os }}-v1-node-module-cache-${{ hashFiles('rslib-bridge/anki/package.json') }}
+          restore-keys: |
+            ${{ runner.os }}-v1-node-module-cache-
+            ${{ runner.os }}-v1-node-module-cache
```

These two lines look odd to me (here and in all the .yml files) - is this intentional? Or should they all be

```suggestion
            ${{ runner.os }}-v1-node-module-cache
```

---

Also just as a side note, I do a lot of github actions .yml work and this is not something we can change here easily - but I wish these blocks were not duplicated everywhere. I find code sharing / code abstraction in the github actions yml blocks to be kind of annoying and I haven't figured out a direction I want to go in yet.

Please don't take any action here - I want this PR to go through really soon :-) - but I know you are doing a lot of yml work now too, if you can think of a good + easy way to de-duplicate these "cache the same thing in all our workflows, hashing the same file, with the same keys" stuff, that would be great as a follow-up. Not a priority though, just a thought. Cheers

---

I have reduce the duplication. I concatenated the hash of `defs.bzl` and `package.json`. 
The cache needs to be included in each yml file, so I think, may be no further de-duplication can be done.